common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -252,4 +252,29 @@ public class RssClientConf {
           .defaultValue(false)
           .withDescription(
               "Whether to support rss client block send failure retry, default value is false.");
+
+  public static final ConfigOption<Integer> RSS_CLIENT_REMOTE_MERGE_FETCH_INIT_SLEEP_MS =
+      ConfigOptions.key("rss.client.remote.merge.fetch.init.sleep.ms")
```

Comment:
`rss.client.remote.merge.fetch.init.sleep.ms` -> `rss.client.remote.merge.fetchInitSleepMs`

Reply:
OK, I have change this and other config.